lib/utils.js

```diff
@@ -7,6 +7,12 @@ function isPlainObject(value) {
 }
 exports.isPlainObject = isPlainObject;
 
+// Support dynamic require within webpack bundle.
+if (typeof __non_webpack_require__ === 'undefined') {
+  var __non_webpack_require__ = require;
```

Comment:
So utils.js is transformed from:
```js
// Support dynamic require within webpack bundle.
if (typeof __non_webpack_require__ === 'undefined') {
  var __non_webpack_require__ = require;
}
exports.require = require;
```

```
to
```js
// Support dynamic require within webpack bundle.
if (false) {
  var __non_webpack_require__ = require;
}
exports.require = require;
```